src/test/components/MarkerChart.test.js

```diff
@@ -409,6 +409,7 @@ describe('MarkerChart', function() {
       rightClick(findFillTextPosition('UserTiming A'));
 
       expect(getContextMenu()).toHaveClass('react-contextmenu--visible');
+      expect(getContextMenu()).toMatchSnapshot();
```

Comment:
I realized we missed that, which made it a bit more difficult to see the changes thanks to the snapshots. So I added it :)

Reply:
Thanks for adding it!